machine.go

```diff
@@ -332,6 +342,11 @@ func (m *Machine) startVMM(ctx context.Context) error {
 		os.Remove(m.cfg.LogFifo)
 		os.Remove(m.cfg.MetricsFifo)
 		errCh <- err
+
+		// Notify subscribers that there will be no more values.
+		// When err is nil, two reads are performed (waitForSocket and close exitCh goroutine),
+		// second one never ends as it tries to read from empty channel.
+		close(errCh)
```

Comment:
It looks like line 375 can also write to this channel.  I think we need strong assurance that the channel won't be closed then, as writing to a closed channel causes a panic.

Reply:
Ignore me...